README.md

```diff
@@ -45,13 +45,19 @@ function render(string $markdown): string {
 }
 ```
 
-For complete compatibility with GitHub Flavored Markdown, support for embedded HTML must be enabled; it is disabled
-by default as a security precaution.
+FBMarkdown currently supports three types of Markdown sources, with plans to expand: trusted, sponsored, and user-generated content.
 
-```Hack
-$ctx = (new Markdown\ParserContext())->enableHTML_UNSAFE();
-$ast = Markdown\parse($ctx, $markdown);
-```
+- __Trusted content mode__: For the parser, (a) embedded HTML is enabled, and (b) all URI schemes are enabled and will be parsed as links. For the renderer, `<img` tags are processed normally. In other modes, the `<img` tag is by default part of the HTML tags denylist, which we filter out. 
+
+- __Sponsored mode__: For the parser, a default set of URI schemes is enabled (`http`, `https`, `irc`, and `mailto`). For the renderer, `rel="nofollow ugc"` is added to all links.
+
+- __User-generated content__: For the parser, (a) embedded HTML is disabled, and (b) all URI schemes are disabled. For the renderer, `rel="nofollow ugc"` is added to all links.
```

Comment:
... "all HTML is disabled, as are links and images regardless of schemes. If links are re-enabled, `rel="nofollow ugc"` will be added to all links.